### ISSUE TITLE
How do you get the SCSS to compile?

### ISSUE THREAD

**Issue by taserak** [ConnectThink/WP-SCSS#164]
I used to use 
define('WP_SCSS_ALWAYS_RECOMPILE', true);

to force it, but is that the only/expected way?
Is there a compile button somewhere?

**Comment by mhbapcc** [comment]
No, unfortunately there is no button for recompiling. I prefer to always recompile for logged in admins (or a specific user/admin), like this...
```
function wpscss_compile_options() {
    if ( !empty( $user = wp_get_current_user() ) ) {
        if ( !is_admin() && $user->user_login == 'SPECIFIC_ADMIN' ) {
            define( 'WP_SCSS_ALWAYS_RECOMPILE', true );
        }
    }
}
add_action( 'init', 'wpscss_compile_options' );
```
Additionally, you'll save time if you don't have to press a button every time.

**Comment by shadoath** [comment]
Introduced an option in the WP-SCSS settings to enable `always recompile` without needing to define `WP_SCSS_ALWAYS_RECOMPILE`.  Now available in version 2.1.1

**Comment by shadoath** [comment]
@mandrasch Yes, you do understand it correctly. With that flag turned on it will recompile the SCSS into CSS on every page load. This should only be used in development.

If you have the Development skills, open a PR with the changes in the [options.php](https://github.com/ConnectThink/WP-SCSS/blob/master/options.php#L227) file for the new wording.

**Comment by mandrasch** [comment]
> @mandrasch Yes, you do understand it correctly. With that flag turned on it will recompile the SCSS into CSS on every page load. This should only be used in development.
> 
> If you have the Development skills, open a PR with the changes in the [options.php](https://github.com/ConnectThink/WP-SCSS/blob/master/options.php#L227) file for the new wording.
> 
> Thank you for your gratitude, it provides fuel for me to keep working on this.

Thought about this and imho a "Clear compiled files"-button would be more descriptive and easier to use? This would also avoid that users mistakenly leave "Always recompile" flag activated on production sites? 🤔

I could try a PR if you think this would be a good option. 

(From what I saw you implemented a timestamp compare check https://github.com/ConnectThink/WP-SCSS/blob/master/class/class-wp-scss.php#L190. So if the corresponding files in via https://github.com/ConnectThink/WP-SCSS/blob/master/class/class-wp-scss.php#L87 aren't there, it should recompile. I guess the easiest option would be a "delete all css files from the output folder", but users must be aware that they shouldn't put custom files in the output directory)
